### PR TITLE
chore(player) - added documentation for text tracks in the Player sec…

### PR DESCRIPTION
**Pull request by guysarfatty**
…tion

## Checklist

* [ ] PR is scoped to one task
* [ ] Tests are included
* [ ] Documentation is included

* [ ] This PR is not making any UI change
* [ ] This PR is making a UI change
  * [ ] Screenshot(s) included

## QA :

* required test cases:
* specific apps / plugins to test :

### CODE REVIEW

**Comment by liviur** [comment]
@guysarfatty please add a link to it from the player guides like there are for the ads and analytics

**Comment by liviur** [comment]
and also link to it from the DSP docs

Thanks

**Comment by guysarfatty** [comment]
@liviur I understood where you requested to add a link on the Player doc.. 

But where exactly in the DSP docs do you think should be a link to the text tracks doc?

**Comment by liviur** [comment]
@guysarfatty - https://developer.applicaster.com/Zapp-Pipes/5.-Feed-API.html

File: book/Zapp-Pipes/5.-Feed-API.md [hunk 1]
```diff
@@ -214,14 +214,24 @@ The media_item element MUST be contained inside a  media_group element.
...
 
 This field, under the extensions object, defines the time in which the component that holds this Feed as a datasource needs to be refreshed. For example, if your Feed contains program which ends in 30 minutes and you want to remove this program from the screen after it's over - the provider can define the `expires_at` value to be now + 30 minutes (date formatted string with this format `"yyyy/MM/dd HH:mm:ss Z"`). Then the component will trigger the provider's load method again when the time comes.
 
 #### free
 
 The `free` key can be defined on the extensions object to indicate whether an item is free or not. The value can either be a String representation of a Boolean (true/false) or a Boolean. If the extension key is not defined or a bad value was supplied the default item behavior will be as free.
 
+#### video_ads
+The datasource can provide information about video ads to display with the video item.
+The video ads configuration should then be handled by the Video Player to display ads before, after or during the video playback.
```
Comment (liviur) [inline book/Zapp-Pipes/5.-Feed-API.md]: Lets change it from `configuration should then be handled` to `configuration will be handled by the video player if it supports displaying video ads`

File: book/Zapp-Pipes/5.-Feed-API.md [hunk 2]
```diff
@@ -214,14 +214,24 @@ The media_item element MUST be contained inside a  media_group element.
...
+To learn more about the suggested protocol to integrate video ads into the extensions section, click [here](https://developer.applicaster.com/player/ads-in-datasource.html).
```
Comment (liviur) [inline book/Zapp-Pipes/5.-Feed-API.md]: please change suggested protocol to optional

File: book/Zapp-Pipes/5.-Feed-API.md [hunk 3]
```diff
@@ -214,14 +214,24 @@ The media_item element MUST be contained inside a  media_group element.
...
+
+#### text_tracks
+The datasource can provide information about text tracks (captions / subtitles) to display on top of the video.
+The text tracks configuration should then be handled by the Video Player.
```
Comment (liviur) [inline book/Zapp-Pipes/5.-Feed-API.md]: same comment - if it supports

File: book/Zapp-Pipes/5.-Feed-API.md [hunk 4]
```diff
@@ -214,14 +214,24 @@ The media_item element MUST be contained inside a  media_group element.
...
+To learn more about the suggested protocol to integrate text tracks into the extensions section, click [here](https://developer.applicaster.com/player/text-tracks-from-dsp.html).
```
Comment (liviur) [inline book/Zapp-Pipes/5.-Feed-API.md]: same comment here - optional

**Comment by lomudi** [comment]
@snooky23 you can review this while Liviu is out of town?

**Pull request merged**